src/Aspire.Hosting.Milvus/MilvusServerResource.cs

```diff
@@ -40,7 +40,7 @@ public MilvusServerResource(string name, ParameterResource apiKey) : base(name)
     /// </summary>
     public ReferenceExpression ConnectionStringExpression =>
        ReferenceExpression.Create(
-            $"Endpoint={PrimaryEndpoint.Property(EndpointProperty.Url)};Key={ApiKeyParameter}");
+            $"Endpoint={PrimaryEndpoint.Property(EndpointProperty.Url)};Key=root:{ApiKeyParameter}");
```

Comment:
This is a small breaking change to existing apps. But I think it makes sense going forward.